saleor/plugins/webhook/utils.py

```diff
@@ -201,3 +255,25 @@ def clear_successful_delivery(delivery: "EventDelivery"):
         delivery.delete()
         if payload_id:
             EventPayload.objects.filter(pk=payload_id, deliveries__isnull=True).delete()
+
+
+DEFAULT_TAX_CODE = "UNMAPPED"
+DEFAULT_TAX_DESCRIPTION = "Unmapped Product/Product Type"
+
+
+def get_current_tax_app() -> Optional[App]:
```

Comment:
Maybe we should split this file and create few smaller ones for each sync webhook? All tax-webhook related things could go to `saleor/plugins/webook/tax_sync_webhook.py`, and payment-related to `payment_sync_webhook.py` ? We can also do it later but I think now it's all mixed here.

Reply:
Great idea but I prefer to move this to a separate task. I will create a proper task for that.

Reply:
https://app.clickup.com/t/2pxnxhg